platform/server-bundle/src/main/java/org/motechproject/server/web/dto/ResetViewData.java

```diff
@@ -23,19 +23,19 @@ public void setResetForm(ResetForm resetForm) {
         this.resetForm = resetForm;
     }
 
-    public boolean isResetSucceed() {
+    public boolean isIsResetSucceed() {
```

Comment:
Did you mean to make this change? `isResetSucceed` makes more sense to me, so wondering whether it maybe got changed along with `setIsResetSucceed` below by mistake.

Reply:
will change it back -- looks like this was tried along with other changes to fix PIT's - but eventually not needed.